requirements-openvino.txt

```diff
@@ -1,8 +1,8 @@
 # Common dependencies
 -r requirements-common.txt
 
-# OpenVINO dependencies
-torch >= 2.1.2
-openvino ~= 2024.4.0
-openvino-tokenizers[transformers] ~= 2024.4.0
-optimum-intel[openvino] >= 1.19.0
+torch == 2.4.0 #  should be aligned with "common" vLLM torch version
+openvino >= 2024.4.0 # since 2024.4.0 both CPU and GPU support Paged Attention
+
+optimum @ git+https://github.com/huggingface/optimum.git@main # latest optimum is used to support latest transformers version
+optimum-intel[nncf] @ git+https://github.com/huggingface/optimum-intel.git@main # latest optimum-intel is used to support latest transformers version
```

Comment:
This seems.. not good to point to the main branch. Is there another option here to at least point to a less-updated stable branch?

Reply:
I know, but optimum released version is always up to 1 month behind latest transformer version and users are constantly facing this issue.

Reply:
I see. With vLLM's heavy reliance on torch and the push to use the latest features available from torch.compile, we need to follow the bleeding edge of pytorch development